molgenis-data/src/main/java/org/molgenis/DatabaseConfig.java

```diff
@@ -65,9 +66,15 @@ public DataSource dataSource()
 	}
 
 	@Bean
+	public IdGenerator idGenerator()
```

Comment:
why is this needed? there is an autowired idGenerator available in this class right?